tests/test_filtsmooth/test_gaussfiltsmooth/test_kalmanposterior.py

```diff
@@ -169,16 +169,17 @@ def test_sampling_shapes(samples, posterior, locs, size):
     assert samples.shape == expected_size
 
 
-def test_sampling_chi_squared(samples, posterior, locs, size):
-    """Returned samples are approximately Gaussian."""
-
-    # what is computed below in here???
-    all_chi_squareds = [
-        chi_squared_statistic(sample, posterior(locs).mean, posterior(locs).cov)
-        for sample in samples
-    ]
-    chi_squared = np.mean(all_chi_squareds)
-
-    # This range is quite large, but a failing tests here would at least
-    # unveil strong deviations...
-    assert 0.01 < chi_squared < 100.0
+#
+# def test_sampling_chi_squared(samples, posterior, locs, size):
+#     """Returned samples are approximately Gaussian."""
+#
+#     # what is computed below in here???
+#     all_chi_squareds = [
+#         chi_squared_statistic(sample, posterior(locs).mean, posterior(locs).cov)
+#         for sample in samples
+#     ]
+#     chi_squared = np.mean(all_chi_squareds)
+#
+#     # This range is quite large, but a failing tests here would at least
+#     # unveil strong deviations...
+#     assert 0.01 < chi_squared < 100.0
```

Comment:
what does this test do? Do you know? I am pretty sure it does at least not do what it claims to do. I'd delete it, but please tell me your opinion.

Reply:
Did I write that?? If yes I have 0 memories of it.. :neutral_face: 

I understand it roughly as "compute the chi2 statistics and check that it lies in the 99% interval" (probably not 99% but just some very large one).

Reply:
but is this really the chi2 statistic??? 

In any case, I removed it. Wont hurt us I reckon.

Reply:
I think it is. But I might be wrong :sweat_smile: